src/color-picker/color-picker.scss

```diff
@@ -235,9 +240,10 @@ $iui-hover-box-shadow: 0 0 1px $iui-xxs + 1 rgba(t(iui-color-foreground-body-rgb
 }
 
 @mixin iui-color-dot {
+  --iui-color-dot-inset: 0 0;
+
   position: absolute;
-  top: var(--top, 0%);
-  left: var(--left, 0%);
+  inset: var(--iui-color-dot-inset);
```

Comment:
inset not supported in IE... wont be too big change?

Reply:
well, CSS variables are also not supported in IE, so it doesn't matter if we use inset

Reply:
good point :D